puppy.py

```diff
@@ -28,4 +29,177 @@ def __init__(self, behavior_matches):
         # the puppy.
         self.touch_sensor = TouchSensor(Port.S1)
 
+    def walk_in_place(puppy, speed=100):
+        puppy.right_leg_motor.run_angle(speed, 40, wait=False)
+        puppy.left_leg_motor.run_angle(speed, -40, wait=True)
+        puppy.right_leg_motor.run_angle(speed, -40, wait=False)
+        puppy.left_leg_motor.run_angle(speed, 40, wait=True)
+        puppy.right_leg_motor.run_angle(speed, -40, wait=False)
+        puppy.left_leg_motor.run_angle(speed, 40, wait=True)
+        puppy.right_leg_motor.run_angle(speed, 40, wait=False)
+        puppy.left_leg_motor.run_angle(speed, -40, wait=True)
+
+    def sit(left_leg_motor, right_leg_motor):
```

Comment:
argument here should be sit(self)
inside of the function, it should be self.left_leg_motor and self.right_leg_motor